packages/frontend-api/apiary.apib

```diff
@@ -2020,6 +2020,40 @@ Returns customer user new personal data
         }
     }
 
+## Register a new customer user [/graphql{?register}]
+
+### POST [POST]
+
+Register and login a new customer user and return access and refresh tokens
+
+- Request (application/json)
+
+    - Body
+        mutation {
+            Register(input: {
+                firstName: "John"
+                lastName: "Doe"
+      	        email: "new-no-reply@shopsys.com"
+      	        password: "user123"
+      	        privacyPolicy: true
```

Comment:
I'm not sure whether it is necessary to send privacyPolicy. 
In the end, it does not guarantee that the user will agree to the terms (it's based on the implementation of the client app)

Reply:
I am not sure either. That's the reason of a standalone commit. I followed constraints implemented in registration form. 
User has to agree with privacy policy, when he registers within the registration form in SSFW. This is checked on the server side.

So it will be different in FE API than, if I remove this check. Is it ok ?

Reply:
Commit has been removed.